cafebabel/articles/translations/views.py

```diff
@@ -0,0 +1,69 @@
+from http import HTTPStatus
+
+from flask import Blueprint, abort, flash, redirect, render_template, request
+from flask_login import current_user, login_required
+from mongoengine import errors
+
+from ... import app
+from ..models import Article
+from .models import Translation
+
+translations = Blueprint('translations', __name__)
+
+
+@translations.route('/new/', methods=['get', 'post'])
+@login_required
+def create():
+    original_article = request.args.get('original')
+    language = request.args.get('lang')
+    if not original_article:
+        abort(HTTPStatus.BAD_REQUEST, 'You must specify a valid article.')
+    if not language or language not in dict(app.config['LANGUAGES']):
+        abort(HTTPStatus.BAD_REQUEST, 'You must specify a valid language.')
+    article = Article.objects.get_or_404(id=original_article)
+
+    if request.method == 'POST':
```

Comment:
We should try to be uniform on this.
Eventually we remove the other views where _get_ and _post_ methods are different routes and make a check in the view if you'd rather go this road on others files too.

Reply:
Isn't it what I did?

Reply:
You did it this way here, but we have it the other way in the other views.

Reply:
Well, I don't get it, I made it uniform everywhere in that PR.

Reply:
Right, spots I had in mind may have been rewritten. My bad.